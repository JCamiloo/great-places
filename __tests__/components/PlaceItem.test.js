import React from "react";
import PlaceItem from "../../components/PlaceItem";
import renderer from "react-test-renderer";

describe("PlaceItem component tests", () => {
  test("PlaceItem component renders correctly", () => {
    const tree = renderer.create(<PlaceItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
