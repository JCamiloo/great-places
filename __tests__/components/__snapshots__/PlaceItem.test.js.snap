// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceItem component tests PlaceItem component renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "borderBottomColor": "#ccc",
      "borderBottomWidth": 1,
      "flexDirection": "row",
      "opacity": 1,
      "paddingHorizontal": 30,
      "paddingVertical": 15,
    }
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "backgroundColor": "#ccc",
        "borderColor": "#fc9208",
        "borderRadius": 35,
        "borderWidth": 1,
        "height": 70,
        "width": 70,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "justifyContent": "center",
        "marginLeft": 25,
        "width": 250,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
          "fontSize": 18,
          "marginBottom": 5,
        }
      }
    />
    <Text
      style={
        Object {
          "color": "#666",
          "fontSize": 16,
        }
      }
    />
  </View>
</View>
`;
